js/about/preferences.js

```diff
@@ -722,7 +722,17 @@ class TabsTab extends ImmutableComponent {
 class PaymentsTab extends ImmutableComponent {
   constructor () {
     super()
+    this.state = {
```

Comment:
Can you help me understand why? Will I also need to create and pass down actions for updating those keys as well? Thanks!